python/cudf/dataframe/series.py

```diff
@@ -1224,6 +1224,15 @@ def mean_var(self, ddof=1):
     def sum_of_squares(self, dtype=None):
         return self._column.sum_of_squares(dtype=dtype)
 
+    def round(self, decimals=0):
+        """
+        """
+        if self.empty:
+            return self
+
+        rounded = cudautils.apply_round(self.data.to_gpu_array(), decimals)
+        return Series(rounded, name=self.name, index=self.index)
```

Comment:
Need to pass `self.nullmask` through here as well if it exists

Reply:
Passing the nullmask now that this is operating at the Column level.